### PR TITLE
[CR] Fixed the weight of some chemical items.

### PR DESCRIPTION
**Pull request by ThinkInvis**
Also added a comment to avoid this problem in the future.
Instead of just dividing the weight by 100, 50, etc., new weights were calculated using the real density for each chemical. Calculations made assuming 1 'volume' is around 0.25 liters.

CR: This is my first attempt at changing C:DDA. Did I make the right assumptions for weight (specifically: are decimals okay) and use git correctly?

### CODE REVIEW

**Comment by KA101** [comment]
Well, you got your volume exactly right.  Nice assumption and use thereof.

I'm not sure re decimals--not much precedent--but they didn't break Jenkins so that's a good sign.

And your Git use looks OK on this end: we prefer (as in, if we catch you not doing this we'll make it  a problem) that you have a local repo and work from that.  Looks like you did.

Queued.

**Comment by DavidKeaton** [comment]
I'm liking this. I'm always up for correctness to RL counterpart. Ace work chief!

File: data/json/items/chemicals_and_resources.json [hunk 1]
```diff
@@ -367,7 +367,7 @@
     "description" : "Several crystals of aluminium sulphate. Historically, an agent for purifying water and as molluscicide, due to the fact that it degenerates their slime. Just before the Cataclysm, its main uses were for adjusting the acidity of the soil and as an additive to baking powder. If mixed with a strong acid, it could be used to saturate the soil with aluminium, causing all but the most resistant of plants to wither and die.",
     "material" : "powder",
     "volume" : 1,
-    "weight" : 350,
+    "weight" : 13.35,
```
Comment (KA101) [inline data/json/items/chemicals_and_resources.json]: OK, so this should weigh 0.65 kg for 50 units?

Comment (ThinkInvis) [inline data/json/items/chemicals_and_resources.json]: 0.67 (0.6675), actually. C:DDA must not handle decimals too well.

**Comment by KA101** [comment]
Close enough IMO.

Welcome to the open source community, ThinkInvis.  :-)

File: data/json/items/chemicals_and_resources.json (hunk 1 above)
Comment (BevapDin) [inline data/json/items/chemicals_and_resources.json]: > C:DDA must not handle decimals too well.

It does handle them. However, the item weight is loaded and stored as integer. Everything behind the '.' is effectively ignored, sorry.

**Pull request merged**